container.sh

```diff
@@ -0,0 +1,26 @@
+ENGINE=docker
+
+if ! which "$ENGINE" &>/dev/null ; then
+    ENGINE=podman
+fi
+
+if ! which "$ENGINE" &>/dev/null ; then
+    echo "No docker or podman in PATH" >&2
+    exit 1
+fi
+
+read -d '' script <<EOF
+cd /app/nix-user-chroot
```

Comment:
I would prefer only to bind mount the only the application source code.

```suggestion
cd /app
```

Reply:
Yep, yep, that was leftover